crates/vm-superio/src/serial.rs

```diff
@@ -914,4 +1013,110 @@ mod tests {
         // THR empty interrupt was raised nevertheless.
         assert_eq!(iir & IIR_THR_EMPTY_BIT, IIR_THR_EMPTY_BIT);
     }
+
+    #[test]
+    fn test_serial_state_constructor() {
+        let state = SerialState {
+            baud_divisor_low: DEFAULT_BAUD_DIVISOR_LOW,
+            baud_divisor_high: DEFAULT_BAUD_DIVISOR_HIGH,
+            interrupt_enable: DEFAULT_INTERRUPT_ENABLE,
+            interrupt_identification: DEFAULT_INTERRUPT_IDENTIFICATION,
+            line_control: DEFAULT_LINE_CONTROL,
+            line_status: DEFAULT_LINE_STATUS,
+            modem_control: DEFAULT_MODEM_CONTROL,
+            modem_status: DEFAULT_MODEM_STATUS,
+            scratch: DEFAULT_SCRATCH,
+            in_buffer: Vec::new(),
```

Comment:
We can unify this test with `test_from_state_with_too_many_bytes`.

Reply:
ACK